gym/wrappers/resize_observation.py

```diff
@@ -39,7 +39,16 @@ def __init__(self, env: gym.Env, shape: Union[tuple, int]):
 
         self.shape = tuple(shape)
 
-        obs_shape = self.shape + self.observation_space.shape[2:]
+        assert isinstance(
+            env.observation_space, Box
+        ), f"Expected the observation space to be Box, actual type: {type(env.observation_space)}"
+        assert len(env.observation_space.shape) == 3 and env.observation_space.shape[
```

Comment:
This seems irrelevant to typing, right? Is this actually necessary for this wrapper to work, or is there any conceivable situation where this could break otherwise functional code?

Reply:
You right I will remove, the assert is not necessary for the PR but I think true however could cause more issues